### ISSUE TITLE
Basic example of README file not working with create react app

### ISSUE THREAD

**Issue by ImanMh** [visgl/react-map-gl#946]
I created an empty project with CRA: 

```
npx create-react-app cra-react-map-gl
cd  cra-react-map-gl
npm run eject
npm i react-map-gl
```

then I copy paste the example code in README to the App.js and I get this error: 

```
Uncaught DOMException: Failed to execute 'postMessage' on 'DedicatedWorkerGlobalScope': ArrayBuffer at index 100 could not be transferred.
    at a (blob:http://localhost:3000/6745b5d0-881f-486a-b1b6-2802e4a62c6d:12682:22)
    at MessagePort.Ou.MessageChannel._channel.port2.onmessage (blob:http://localhost:3000
```
here is a full screenshot of the error: 

<img width="937" alt="Screen Shot 2019-11-24 at 11 45 09 PM" src="https://user-images.githubusercontent.com/4482199/69500874-75e62100-0f14-11ea-9ede-a1c6b71516b9.png">

This is a little bit strange that why react version of mapbox-gl is not working with the most popular react boilerplate. Am I doing something wrong?

**Comment by RobJacobson** [comment]
Could you try posting your code to a codesandbox.io workspace, and then share that?  I use react-map-gl with create-react-app every day, so there may be something unexpected going on in your code.

**Comment by RobJacobson** [comment]
You can also check out this "controls" example with markers and popups.  It's an example from the react-map-gl team that I modified so that it uses create-react-app as the baseline.

https://github.com/RobJacobson/react-map-gl-examples-cra

**Comment by ImanMh** [comment]
Sorry guys, I think I forgot to close this issue after I figured out the problem. After a few hours of struggling I found out that this was only happening on Google Chrome Canary. I don't know that it's a bug in the alpha version of chrome and they will fix it or it's gonna happen in stable version of chrome in the future as well. If you think it's something worth taking a look let me know and I'll write the example of this project in a sandbox.
